en/02_Developer_Guides/03_Forms/01_Validation.md

```diff
@@ -71,71 +71,94 @@ In this example we will be required to input a value for `Name` and a valid emai
 
 > [!NOTE]
 > Each individual [FormField](api:SilverStripe\Forms\FormField) instance is responsible for validating the submitted content through the
-> [FormField::validate()](api:SilverStripe\Forms\FormField::validate()) method. By default, this just checks the value exists. Fields like `EmailField` override
-> `validate` to check for a specific format.
+> [FormField::validate()](api:SilverStripe\Forms\FormField::validate()) method. Fields like [`EmailField::validate()`](api:SilverStripe\Forms\EmailField::validate()) check for a specific format.
```

Comment:
`EmailField::validate()` is not a field - either change back to the wording used before, or update reflect that you're not talking about the field, but the method _on_ the field.